backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepository.java

```diff
@@ -51,6 +51,9 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(LiveOktaRepository.class);
 
+  private static final String FILTER_TYPE_EQ_OKTA_GROUP =
```

Comment:
There are a couple of other places where we will be able to use this filter with listGroups to get Okta to perform type filtering instead of us checking in `.stream().filter()`.

Reply:
Do we actually ever get groups of other types?

Reply:
We shouldn't, but I would defer to a bit more investigation before removing the filtering.